src/test/java/org/tarantool/AbstractTarantoolConnectorIT.java

```diff
@@ -180,27 +181,49 @@ protected TarantoolConnection openConnection() {
         }
     }
 
-    protected List<?> consoleSelect(String spaceName, Object key) {
-        StringBuilder sb = new StringBuilder("box.space.");
-        sb.append(spaceName);
-        sb.append(":select{");
+    private void appendBigInteger(StringBuilder sb, BigInteger value) {
+        sb.append(value);
+        sb.append(value.signum() >= 0 ? "ULL" : "LL");
+    }
+
+    private void appendKey(StringBuilder sb, Object key) {
         if (List.class.isAssignableFrom(key.getClass())) {
-            List parts = (List)key;
+            List parts = (List) key;
             for (int i = 0; i < parts.size(); i++) {
                 if (i != 0)
                     sb.append(", ");
                 Object k = parts.get(i);
-                if (k.getClass().isAssignableFrom(String.class)) {
+                if (k instanceof BigInteger) {
+                    appendBigInteger(sb, (BigInteger) k);
+                } else if (k.getClass().isAssignableFrom(String.class)) {
```

Comment:
Strange check that a class of variable 'k' must be an ancestor of String. Maybe it's better to use instanceof String in this case ( or instanceof CharSequece depending of what type of variable 'k' is supposed in code below)

Reply:
Yep, this check is strange one. I had replaced it with `instanceof String`.